Sample Data/Custom/Sonrai_CL.json

```diff
@@ -0,0 +1,96 @@
+[
```

Comment:
File name for sample data should match with custom table name. So in our case it should be **Sonrai_Tickets_CL** .

Reply:
Renamed the file to Sonrai_Tickets_CL.json